tests/readrows_test.go

```diff
@@ -447,3 +447,51 @@ func TestReadRows_NoRetry_MultipleRowRanges(t *testing.T) {
 	res := doReadRowsOp(t, server, &req, nil)
 	assert.Len(t, res.Row, 4)
 }
+
+// TestReadRows_NoRetry_OpenEndUnspecifiedStart tests that the client can request
+// a row range with an open end key and no start key.
+func TestReadRows_NoRetry_OpenEndUnspecifiedStart(t *testing.T) {
+	type testRow struct {
+		key, cf string
```

Comment:
"cf" sounds like "Column Family", can it be renamed to something like "value"?

Reply:
OH, that's what I thought they were. Sure, I can change this.